AutomatedTesting/Registry/asset_cache_server_settings.setreg

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Should this file go in the `<ENGINE_ROOT>\Registry` so that all projects would get these settings?  Or are these overrides just for asset cache settings for the `AutomatedTesting` project?

Reply:
This is a project specific setreg file, so it does not make sense to add it to the engine. This file gets written out when the UX "Submit" button is pressed, so it should not write it to an Engine file/path.